core/readme.md

```diff
@@ -0,0 +1,33 @@
+# Contributing CAPs
+
+ 1. Review [CAP-0001](cap-0001.md).
```

Comment:
This will be "Bump Sequence" right? By "review" do you mean check how regular CAP looks like?